charts/signoz/tests/signoz-instance_env_test.yaml

```
@@ -0,0 +1,139 @@
+      - notContains:
+          path: spec.template.spec.containers[0].env
+          content:
+            name: SIGNOZ_EMAILING_SMTP_HOST
+
+  # --- Test Case 2: SMTP enabled with the new 'env' convention ---
+  - it: should render SMTP variables correctly using the new 'env' convention
+    set:
+      signoz:
+        env:
+          signoz_emailing_enabled: true
+          signoz_emailing_smtp_address: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)